README.md

```diff
@@ -6,6 +6,54 @@
 Rust SDK for the Solana blockchain, used by on-chain programs and the Agave
 validator.
 
+## Upgrading from v2 to v3
+
+### solana-sdk
+
+The following modules have been removed, please use their component crates
+directly:
```

Comment:
Wonder if it would be useful for this to be links to crate.io or docs.rs for each individual crate.